### DISCUSSION TITLE
Should I add a web app at Firebase when making a mobile game?

### DISCUSSION THREAD

**Discussion by Gencreezo** [GodotNuts/GodotFirebase#409] [Q&A] [answered]
In the Installation Guide, it is written to add a Web App at Firebase. Is this also the recommended way when I'm planning to make a mobile only game for android and perhaps ios later on?

**Comment by WolfgangSenff** [comment, marked as answer]
You can use Android and iOS, it's just a little more annoying because we don't use the files that come from their creation for our environment variables - we just use the .env file, and it's easier if we just want one file for all platforms. Given that, I prefer just to make it a web app, especially since you can't conditionally export two different .env files that still apply based on the platform you're on. Although I suppose I could try to implement that.
